packages/terra-action-header/tests/jest/ActionHeader.test.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-extraneous-dependencies, no-alert */
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import Button from 'terra-button';
+import ActionHeader from '../../src/ActionHeader';
+import messages from '../../translations/en-US.json';
+
+const locale = 'en-US';
+
+describe('ActionHeader', () => {
+  // Snapshot Tests
+  it('should render a default action header', () => {
+    const actionHeader = render(
+      <IntlProvider locale={locale} messages={messages}>
```

Comment:
We should drop the `IntlProvider` from these tests and test `ActionHeader` as itself. We can add intl context with [these setup configs](https://github.com/cerner/terra-core/blob/master/packages/terra-tabs/tests/jest/intl-context-setup.js), and we can provide the intl context [like this](https://github.com/cerner/terra-core/blob/master/packages/terra-tabs/tests/jest/Tabs.test.jsx#L8)

Reply:
Is that something we should consider moving to terra-toolkit? Seems like multiple tests can benefit from that.

Reply:
Another option potentially is to try shallow rendering. It looks like this component does it with proper messages. https://github.com/cerner/terra-core/blob/master/packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx

Reply:
Updated [here](https://github.com/cerner/terra-core/commit/444fde0838557ce58e76c9faee9b854a6de8d62d).